fleet_packages.json

```diff
@@ -41,6 +41,6 @@
   },
   {
     "name": "security_detection_engine",
-    "version": "8.4.2"
+    "version": "8.4.3-beta.1"
```

Comment:
@terrancedejesus - Is this prerelease version expected to ship to prod? Recall we discussed this over on opt-out of bundling prerelease packages for some packages issue, but that work hasn't been picked up yet.